### PR TITLE
Mohammed connect frontend backend

### PR DESCRIPTION
**Pull request by mojahmed**
code the dream and 1 more page - Personal - Microsoft​ Edge 12_14_2024 3_25_56 PM
code the dream and 1 more page - Personal - Microsoft​ Edge 12_14_2024 3_26_27 PM
code the dream and 1 more page - Personal - Microsoft​ Edge 12_14_2024 3_26_43 PM
code the dream and 1 more page - Personal - Microsoft​ Edge 12_14_2024 3_26_53 PM

I have created the header component which includes navigation and state management for toggling the navigation menu. Here is a summary of the functionality
1-The login system is set up. For now, only users who have registered can access their profile.

2-Upon logging in, users will see their name and a default profile picture

3-The registration functionality is still being fixed, so for testing purposes, you can use the following email to log in.

(example.doe@example.com)password(123456789)

#Make sure to runs the back-end server so you can properly test and see the results.

#I installed (Redux Toolkit-React Router Dom-toast)make sure to run (npm installl)

#The authSlice.js file contains the logic for user login and logout.The loginUser function will make an Axios API call to authenticate the user and update the Redux store accordingly.User authentication state is stored in Redux and persisted in localStorage to maintain sessions across page reloads.When the user logs out, localStorage will be cleared, and the Redux state will be updated to reflect that the user is no longer logged in.

#admindasborad only will be showing on the navbar if the admin logedin

### CODE REVIEW

**Review by tianahR — COMMENTED** [review]
Reviewed and sent my comments on slack channel

File: src/components/header/HeaderLeft.jsx
Comment (AnnaViktorovna) [inline src/components/header/HeaderLeft.jsx:1]: need remove code that we don't use

File: src/pages/Home/home.jsx
Comment (AnnaViktorovna) [inline src/pages/Home/home.jsx:1]: remove previous code

File: src/pages/forms/Register.jsx
Comment (AnnaViktorovna) [inline src/pages/forms/Register.jsx:1]: sorry, I don't understand why all code  looks like comments

**Review by AnnaViktorovna — APPROVED** [review]
I have some questions but approve changes

File: src/pages/forms/Register.jsx
Comment (mojahmed) [reply src/pages/forms/Register.jsx:1]: have you run the code? did you see any error ?

File: src/pages/Home/home.jsx
Comment (mojahmed) [reply src/pages/Home/home.jsx:1]: resolved issues

**Review by vitaliipp — APPROVED** [review]
Great job

**Pull request merged**